### PR TITLE
DFS

### PR DESCRIPTION
**Pull request by mariannauf**

### CODE REVIEW

File: mariannaFervenza/assignments/treesDFS/lc113/lc113.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# if the root is null --> return 0
+# otherwise
+# we check it's left node and add it to the root node
+# then we check left chield again and add it to them sum
+# then we compare with the targetSum
+
+# We need to return a list-of-lists
+# We need to add each of the leaf nodes that were added to the sum
+# to the list-of-lists
+# --> only add to the list if the path is equal to the targetSum
```
Comment (knrodriguez) [inline mariannaFervenza/assignments/treesDFS/lc113/lc113.py:30]: Nice approach.

File: mariannaFervenza/assignments/treesDFS/lc113/lc113.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+# Example
+# -----
+# targetSum = 12
+# Output = [ [ 1, 7, 4 ], [ 1, 9, 2 ] ]
+#      1
+#     / \
+#    7   9
+#  / |   | \
+# 4  5   2  7
+
```
Comment (knrodriguez) [inline mariannaFervenza/assignments/treesDFS/lc113/lc113.py:14]: Missing edge cases.

File: mariannaFervenza/assignments/treesDFS/lc113/lc113.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    root = TreeNode(12)
+    root.left = TreeNode(7)
+    root.right = TreeNode(1)
+    root.left.left = TreeNode(4)
+    root.right.left = TreeNode(10)
+    root.right.right = TreeNode(5)
+    sum = 23
+    print("Tree paths with sum " + str(sum) +
+          ": " + str(find_paths(root, sum)))
+
```
Comment (knrodriguez) [inline mariannaFervenza/assignments/treesDFS/lc113/lc113.py:69]: Missing test case to match edge cases.

**Review by knrodriguez — APPROVED** [review]
